packages/xo-server-sdn-controller/src/index.js

```diff
@@ -666,6 +957,125 @@ class SDNController extends EventEmitter {
     })
   }
 
+  // ---------------------------------------------------------------------------
+
+  async _clearCenterPoolFromNetwork(centerPoolNetwork, poolNetwork, uuid) {
```

Comment:
`uuid` of? Make sure we understand what it is when we read it.
Also, is `uuid` ever used?

Reply:
Only for logging.